hudi-client/src/test/java/org/apache/hudi/table/TestMergeOnReadTable.java

```diff
@@ -80,6 +85,12 @@
 
 public class TestMergeOnReadTable extends HoodieClientTestHarness {
 
+  private HoodieParquetInputFormat roInputFormat;
```

Comment:
We can add similar test-cases for Copy-On-Write table with RO inputformat alone to make it uniform. @satishkotha : Can you look at it as part of this PR or a follow-up ?

Reply:
Thanks for suggestion. I modified one of the existing tests on COW table. Please take a look.